shpc/client/sync.py

```diff
@@ -0,0 +1,31 @@
+__author__ = "Vanessa Sochat"
+__copyright__ = "Copyright 2021-2022, Vanessa Sochat"
+__license__ = "MPL 2.0"
+
```

Comment:
For `os.path.exists` below

```suggestion

import os

```